main.cpp

```diff
@@ -60,14 +60,14 @@ video: Chapter 2 - Part 3
  */
 
 //2)
-void variableDeclarations()
+void variableDeclarations(int myAge = 29, int myHeight = 190, int myWeight = 105, char myGrade = 5, char mySheet = 120, char volume = 11, bool murderIsLegal = false, bool payTaxes = true, bool skyDivingIsFun = false, float chickenWalk = 5.6f, float chickenRun = 7.8f, float gunSpeed = 6.6f, double humanWalk = 6.0, double humanRun = 8.5, double carSpeed = 10)

```

Comment:
Declare these variables INSIDE the function.
Do not add them to the function parameter list.
- [ ] revise